interp/stdarg.ml

```diff
@@ -42,6 +42,8 @@ let wit_var =
 
 let wit_ref = make0 "ref"
 
+let wit_smart_global = make0 "smart_global"
```

Comment:
You need to share the same runtime representation as `wit_ref` using the `~dyn` optional argument.

Reply:
I'm not sure what you mean, can you make an edit suggestion?